codeinterpreterapi/session.py

```diff
@@ -167,10 +167,13 @@ async def output_handler(self, final_response: str) -> CodeInterpreterResponse:
     async def generate_response(
         self,
         user_msg: str,
-        files: list[File] = [],
-        detailed_error: bool = False,
+        files: Optional[list[File]] = None,
+        detailed_error: Optional[bool] = False,
     ) -> CodeInterpreterResponse:
         """Generate a Code Interpreter response based on the user's input."""
+        if files is None:
```

Comment:
this does the same by adding more lines

Reply:
Ok, reverted the change. Indeed in this case the list doesn't seem to change after the call, but from my experience, you can never be too safe with mutable variables :).